py/autoevals/oai.py

```diff
@@ -18,7 +18,12 @@ class OpenAIWrapper:
     RateLimitError: Exception
 
 
+_WRAPPED_OPENAI = False
```

Comment:
Can we avoid the global variable by returning this boolean as part of `prepare_openai`?

Reply:
good call